app/mailers/digest_mailer.rb

```diff
@@ -1,17 +1,20 @@
-class DigestMailer < ActionMailer::Base
-  default from: ENV["MAIL_NOTIFICATION"]
+class DigestMailer < ApplicationMailer
+  default template_path: 'digest_mailer'
+  default template_name: 'digest_mailer'
 
-  # Subject can be set in your I18n file at config/locales/en.yml
-  # with the following lookup:
-  #
-  # en.digest_mailer.daily_digest.subject
-
-  def daily_digest(history, user)
-    @discussions = history.discussions
-    @tutorials = history.tutorials
-    @libraries = history.libraries
-    @new_users_count = User.count - history.users_count
-    @user = user
-    mail to: user.email, subject: 'RubyGameDev.com Daily Digest'
+  def digest_mailer(history, user_id)
+    # @discussions = history.discussions
+    # @tutorials = history.tutorials
+    # @libraries = history.libraries
+    # @new_users_count = User.count - history.users_count
+    # @comment = Comment.find comment_id
+    @internal_post = InternalPost.all
```

Comment:
This logic should be moved into the `DigestHistory` class so you can call `history.posts`. It should only return the posts that are new since the last email was sent. The other methods (`discussions`, `tutorials`, `libraries`) are all old and should be removed.